recipes-multimedia/gstreamer/gstreamer1.0-libav_%.bbappend

```diff
@@ -0,0 +1,2 @@
+# Disable thumb
+ARM_INSTRUCTION_SET_armv7a = "arm"
```

Comment:
if we do it, we should do it for rpb distro only. if it's a bug upstream then it should be dicussed on oe-core list.

Reply:
There are some more details like gdb log snippets under https://projects.linaro.org/browse/MMWG-79 . I would appreciate if people go and take a look.

I am not familiar enough with thumb, and can't tell if the illegal instruction is due to a bug upstream, or due to e.g. the SOC hw bug (my am57xx-evm is fairly early revision, but I am not aware of any errata like that). And I don't have other armv7 boards handy. Hm.. maybe checking db410c with a 32-bit rootfs could make sense before posting to oe-core list?